phpcs.xml

```diff
@@ -1,61 +1,66 @@
 <?xml version="1.0"?>
 <ruleset name="WordPress Coding Standards">
 
-    <!-- Set a description for this ruleset. -->
-    <description>A custom set of code standard rules to check for Nginx helper plugin.</description>
+	<!-- Set a description for this ruleset. -->
+	<description>A custom set of code standard rules to check for Nginx helper plugin.</description>
 
-    <!-- Pass some flags to PHPCS:
+	<!-- Pass some flags to PHPCS:
          p flag: Show progress of the run.
          s flag: Show sniff codes in all reports.
     -->
-    <arg value="psvn"/>
-
-    <!-- Check up to 8 files simultanously. -->
-    <arg name="parallel" value="8"/>
-
-    <!-- Only check the PHP files. JS files are checked separately with JSCS and JSHint. -->
-    <arg name="extensions" value="php"/>
-
-    <!-- Check all files in this directory and the directories below it. -->
-    <file>.</file>
-
-    <!-- Exclude folders. -->
-    <exclude-pattern>*/languages/*</exclude-pattern>
-    <exclude-pattern>*/admin/predis.php</exclude-pattern> <!-- Predis library file -->
-    <exclude-pattern>*/vendor/*</exclude-pattern> <!-- Composer vendor directory  -->
-
-    <!-- Include the WordPress standard. -->
-    <rule ref="WordPress-VIP-Go">
-        <exclude name="WordPress.VIP.FileSystemWritesDisallow.directory_mkdir" />
-        <exclude name="WordPress.WP.AlternativeFunctions.file_system_read_fopen" />
-        <exclude name="WordPress.VIP.FileSystemWritesDisallow.file_ops_is_writable" />
-        <exclude name="WordPress.WP.AlternativeFunctions.file_system_read_fclose" />
-        <exclude name="WordPress.VIP.FileSystemWritesDisallow.file_ops_fwrite" />
-        <exclude name="WordPress.VIP.FileSystemWritesDisallow.directory_rmdir" />
-        <exclude name="WordPress.WP.AlternativeFunctions.file_system_read_fwrite" />
-        <exclude name="WordPress.VIP.FileSystemWritesDisallow.file_ops_unlink" />
-    </rule>
-    <rule ref="WordPress-Docs"/>
-
-    <!-- Verify that the text_domain is set to the desired text-domain.
+	<arg value="psvn"/>
+
+	<!-- Check up to 8 files simultanously. -->
+	<arg name="parallel" value="8"/>
+
+	<!-- Only check the PHP files. JS files are checked separately with JSCS and JSHint. -->
+	<arg name="extensions" value="php"/>
```

Comment:
```suggestion
	<arg name="extensions" value="php,js"/>
```
Could we also include JS along with PHP, How is JS code security checked for the plugin currently?